src/ext-test/java/org/opentripplanner/ext/fares/impl/DefaultFareServiceTest.java

```diff
@@ -185,8 +171,17 @@ void multipleFeeds() {
       fareProductIds
     );
 
-    var resultPrice = result.getFare(FareType.regular);
-    assertEquals(TWENTY_DOLLARS, resultPrice);
+    var legProducts = result.getLegProducts();
+    var firstBusLeg = itin.getTransitLeg(0);
+    assertEquals(
+      "[FareProductUse[id=1d270201-412b-3b86-80f6-92ab144fa2e5, product=FareProduct{id: 'F:airport-to-city-center', name: 'regular', amount: $10.00}]]",
```

Comment:
I would replace all the printouts of `toString` with a direct object comparison.

Reply:
Done in a8762307d6ad1daae85737de58a67d2039e983c7